### ISSUE TITLE
Option for leaving pinned tabs alone

### ISSUE THREAD

**Issue by davidreuss** [Lukargo/Tabvention#1]
Hi,

I have a few tabs pinned which i generally never want to be autoclosed (gmail, issue tracker, etc) -- It would be nice with an option for skipping tabs that are pinned.

**Comment by nobe4** [comment]
Agreed, you can do this by checking the `pinned` property on the tab before deleting it.
ref : https://developer.chrome.com/extensions/tabs#type-Tab
-> https://github.com/Lukargo/Tabvention/pull/2
